src/components/AllServices/AllServicesLink.tsx

```diff
@@ -1,70 +1,63 @@
 import React, { useMemo } from 'react';
 import { Content, ContentVariants } from '@patternfly/react-core/dist/dynamic/components/Content';
+import { Flex } from '@patternfly/react-core/dist/dynamic/layouts/Flex';
+import { FlexItem } from '@patternfly/react-core/dist/dynamic/layouts/Flex';
 import { Icon } from '@patternfly/react-core/dist/dynamic/components/Icon';
 
 import classNames from 'classnames';
 import { matchPath } from 'react-router-dom';
 
-import StarIcon from '@patternfly/react-icons/dist/dynamic/icons/star-icon';
 import ExternalLinkAltIcon from '@patternfly/react-icons/dist/dynamic/icons/external-link-alt-icon';
 
 import ChromeLink from '../ChromeLink';
-import type { AllServicesLink as AllServicesLinkType } from './allServicesLinks';
-import useFavoritePagesWrapper from '../../hooks/useFavoritePagesWrapper';
 import { useAtomValue } from 'jotai';
 import { moduleRoutesAtom } from '../../state/atoms/chromeModuleAtom';
-import { titleToId } from '../../utils/common';
 
-export type AllServicesLinkProps = AllServicesLinkType & { category: string; group?: string };
+interface AllServicesLinkProps {
+  href?: string;
+  title?: string;
+  sectionTitle?: string;
+  bundleTitle?: string;
+  isExternal?: boolean;
+  category?: string;
+  group?: string;
+}
 
-const AllServicesLink = ({ href, title, isExternal, category, group }: AllServicesLinkProps) => {
+const AllServicesLink = ({ href, title, sectionTitle, bundleTitle, isExternal = false }: AllServicesLinkProps) => {
   const moduleRoutes = useAtomValue(moduleRoutesAtom);
   // Find service appId
   const appId = useMemo(() => {
-    return moduleRoutes.find(({ path }) => matchPath(path, href) || matchPath(`${path}/*`, href))?.scope;
+    return moduleRoutes.find(({ path }) => matchPath(path, href ? href : '') || matchPath(`${path}/*`, href ? href : ''))?.scope;
   }, [moduleRoutes, href]);
-  const { favoritePage, unfavoritePage, favoritePages } = useFavoritePagesWrapper();
 
-  const handleFavouriteToggle = (pathname: string, favorite?: boolean) => {
-    if (favorite) {
-      unfavoritePage(pathname);
-    } else {
-      favoritePage(pathname);
-    }
-  };
```

Comment:
This removes the option to favorite item on stage stable as well. Let's add this function back.